package.json

```diff
@@ -11,9 +11,9 @@
   "author": "Aleksis Brezas <abresas@resin.io>",
   "license": "Apache-2.0",
   "scripts": {
-    "pretest": "./node_modules/.bin/coffeelint lib.coffee dockerfile-template.coffee ./test",
-    "test": "./node_modules/mocha/bin/mocha --compilers coffee:coffee-script/register",
-    "prepublish": "./node_modules/.bin/coffee -c lib.coffee dockerfile-template.coffee && (echo '#!/usr/bin/env node' | cat - dockerfile-template.js > dockerfile-template)"
+    "pretest": "coffeelint lib.coffee dockerfile-template.coffee ./test",
+    "test": "mocha --compilers coffee:coffee-script/register",
+    "prepublish": "gulp && (echo '#!/usr/bin/env node' | cat - dockerfile-template.js > dockerfile-template)"
```

Comment:
This should use the [`bin`](https://docs.npmjs.com/files/package.json#bin) property of package.json in order to make the script executable - it handles the execute permission bit on linux, and also generates a .cmd for windows